frontend/jupyter/src/app/pages/form/form-new/form-new.component.ts

```diff
@@ -90,10 +80,6 @@ export class FormDefaultComponent implements OnInit, OnDestroy {
     });
   }
 
-  ngAfterContentChecked() {
-    this.cdr.detectChanges();
-  }
-
```

Comment:
This was added by us. I think it's still needed seee https://github.com/StatCan/jupyter-apis/blame/main/frontend/jupyter/src/app/pages/form/form-default/form-default.component.ts 
and it's not in 1.6 https://github.com/kubeflow/kubeflow/blob/v1.6.1/components/crud-web-apps/jupyter/frontend/src/app/pages/form/form-default/form-default.component.ts

Reply:
Oh yes i think you are right, good catch!